mangaki/mangaki/models.py

```diff
@@ -240,20 +240,43 @@ class WorkTitle(models.Model):
                             blank=True,
                             db_index=True)
 
+    class Meta:
+        unique_together = ('title', 'language')
```

Comment:
I don't think this is correct because we can have works with identical names for different mediums (e.g. there is a "Death Note" anime and a "Death Note" manga).